app/models/workers_scaling/algorithms/resources_usage_minimization.rb

```diff
@@ -68,53 +67,53 @@ def time_constraint_check(predicted, left)
     ##
     # Increases computational power to match required system throughput
     # Does nothing if there are starting Workers already
-    # Uses known infrastructures with highest throughput if limits are not reached
-    # If limits in all known infrastructures are reached, uses random unknown infrastructure
+    # Uses known configurations with highest throughput if limits are not reached
+    # If limits in all known configurations are reached, uses random unknown configuration
     def increase_computational_power
-      LOGGER.debug 'Need to increase computational power'
+      log(:debug, 'Need to increase computational power')
```

Comment:
@dkrol @mpaciore
Nie powiedziałbym, że nic nie wnosi - dodaje do logów komunikat o akcji, która zostanie wykonana w systemie co może pomóc w wyśledzeniu przebiegu kodu w logach w razie jakiejś awarii. Pytanie czy nie można napisać tego komunikatu w taki sposób i w takim miejscu, żeby dawał więcej informacji dlaczego i dla jakiego eksperymentu chcemy wykonać akcję zwiększania computational power?

Co do samego komunikatu - myślę, że lepiej brzmiałoby coś w rodzaju "Increasing computational power", albo "will increase...", bo jest to komunikat o rozpoczęciu wykonywania akcji, a nie potrzebie jej wykonania.

Poza tym chyba zwiększa kod i 1 linijkę a nie dwie (?)

Jeśli już czyścić logi debug, to raczej można by przedyskutować logowanie MongoLocków, które kiedyś sam napisałem.